spec/features/admin_creates_new_neighborhood_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+feature 'admin can create new neighborhood', %Q(
+  As an authenticated user
+  I want to add a neighborhood to Boston
+  So that others can tell me about it (review it).
+) do
+
+  scenario 'admin successfully adds new neighborhood' do
+    user = FactoryGirl.create(:user)
+
+    visit new_neighborhood_path
+    fill_in 'Name', with: "Downtown"
+    select("East", from: "Location")
+    fill_in 'Description', with: "Lame."
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.